index.html

```diff
@@ -4,17 +4,176 @@
+             <a href="#"><img src="images/project.png" class="h-20" alt=""></a>
+             <ul class="flex gap-16 ml-72 text-base font-medium">
+                 <li><a href="About.html" class="text-brandorange">About</a></li>
+                 <li><a href="">Program</a></li>
+                 <li><a href="">Join</a></li>
+                 <li><a href="">Sponsor</a></li>
+                 <li><a href="">News</a></li>
+             </ul>
+             <button type="button" class="ml-28 border-4 border-brandorange text-brandorange p-3">CC Campaign</button>
+          </div>
+        </nav>
     </header>
     <main>
-        <section>
-            <h1>Home pages</h1>
+        <section class="bg-[url('images/conference.jpg')] bg-no-repeat bg-cover md:h-screen">
+            <div class="bg-brandgray bg-opacity-80 md:h-full">
+            <div class="bg-gradient-to-t from-white ">
+            <div class="bg-[url('images/backpattern.png')] bg-no-repeat bg-cover bg-center md:h-screen">
+            <div class="md:hidden">
+                <img src="images/more.png" class="h-14 p-3" alt="humberger menu" onclick="display()" id="menu">
+                <img
+                src="images/close.png"
+                alt="menu"
+                id="menu2"
+                onclick="display()"
+                class="h-12 p-3"
+              />
+            </div>
+            <div class="mob-nav-container" id="mnc">
+                <ul class="mob-nav text-3xl font-black ml-20 text-brandbrown">
+                  <li>
+                    <a href="About.html" onclick="menuClicked()">About</a>
+                  </li>
+                  <li>
+                    <a href="#" onclick="menuClicked()">Join</a>
+                  </li>
+                  <li>
+                    <a href="#" onclick="menuClicked()">Sponsor</a>
+                  </li>
+                  <li>
+                    <a href="#" onclick="menuClicked()">News</a>
+                  </li>
+                </ul>
+              </div>
+            <div class="p-5 md:ml-60 md:pt-40">
+                <h2 class="text-2xl md:text-4xl font-normal text-brandorange ">Hello! Sharing Cardano</h2>
+                <div>
+               <h1 class="bg-clip-text text-transparent bg-[url('images/mainbg.jpg')] bg-cover md:w-1/2 text-4xl md:text-7xl text-brandorange font-black">GLOBAL CARDANO STUDENT SEMINAR 2022</h1>
+            </div>
+               <div class="md:text-lg md:w-1/3 head-qout w-80 text-sm mt-5 border-2 border-white p-5">
+                   <p>Presentations, and demos from Cardano leaders and luminaries hosted in the biggest virtual world ever for a blockchain event.</p>
+               </div>
+               <div class="mt-5 head-sub" >
+                   <h2 class="text-2xl md:text-4xl font-black text-brandbrown">2022.10.15(TUH) ~ 16(FRI)</h2>
+                   <p class="md:text-4xl text-brandbrown">@ Addis Ababa University, AAiT</p>
+               </div>
```

Comment:
Kindly for the headline section, you can update the following
- [ ] You can make your font size for the header a little bit smaller to match the original design.
- [ ] You can give your paragraph more width so it can match the original design.
- [ ] Your design missing two parts the small bold text and the paragraph under it, you can see them on the original design, I see that you have added them but they do not appear.

Your design
![image](https://user-images.githubusercontent.com/35535130/168816257-aa33a880-ce1b-4621-9e52-a91507aaa72b.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/168816436-16778c21-00cb-48c1-9f9e-a6d550d9bebe.png)

Reply:
@keroloussamy Thank you very much for the review and kind feedback, but I am having some issues understanding this because this is how the desktop version looks on my screen.
![image](https://user-images.githubusercontent.com/29539518/168824826-477ffc1b-2763-4221-a169-98b722860689.png)

Reply:
@NatiDeme This difference is because what width do you open the page on, I think it's open fine on the large width but on small widths, there are some parts that don't appear. kindly try to resize your screen and see what will happen.